ontology/dprod/dprod-ontology.ttl

```diff
@@ -1,160 +1,95 @@
 # baseURI: https://ekgf.github.io/dprod/
-# imports: http://purl.org/dc/terms/
-# imports: http://www.w3.org/2004/02/skos/core
-# imports: http://www.w3.org/ns/prov-o#
-# imports: http://www.w3.org/ns/dcat
+# imports: http://www.w3.org/ns/dcat#
 # prefix: dprod
 
-
-@base: <https://ekgf.github.io/dprod/#> .
-@prefix dprod: <https://ekgf.github.io/dprod/#> .
+@base <https://ekgf.github.io/dprod/> .
+@prefix dprod: <https://ekgf.github.io/dprod/> .
 @prefix dcat: <http://www.w3.org/ns/dcat#> .
 @prefix dct: <http://purl.org/dc/terms/> .
-@prefix dc: <http://purl.org/dc/elements/1.1/> .
-@prefix dctype: <http://purl.org/dc/dcmitype/> .
-@prefix foaf: <http://xmlns.com/foaf/0.1/> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix prov: <http://www.w3.org/ns/prov#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix odrl: <http://www.w3.org/ns/odrl/2/> .
-@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+# Since Dublin Core is not an ontology, declare here that the properties used are to be treated as AnnotationProperties in OWL
+dct:description a owl:AnnotationProperty .
+dct:contributor a owl:AnnotationProperty .
+dct:license a owl:AnnotationProperty .
+dct:modified a owl:AnnotationProperty .
 
 dprod:
   a owl:Ontology ;
-  rdfs:label "Data Product Vocabulary"@en ;
-  dc:description "DPROD is an RDF vocabulary designed to facilitate interoperability between data product descriptors." ;
+  dct:title "The Data Product Vocabulary"@en ;
+  rdfs:label "data product ontology"@en ;
+  dct:description "DPROD extends DCAT to enable publishers to describe Data Products and data services in a decentralized way."@en ;
   rdfs:comment
-    "This vocabulary includes terms for the definition of the data product as defined from the Data Product Working Group. The vocabulary utilises the notion of Datasets as defined from DCAT vocabulary"@en ;
-  owl:imports <http://purl.org/dc/terms/> ;
-  owl:imports <http://www.w3.org/2004/02/skos/core> ;
-  owl:imports <http://www.w3.org/ns/prov-o#> ;
-  owl:imports <http://www.w3.org/ns/dcat> ;
-  dct:creator
```

Comment:
Why remove all these contributors?

Reply:
You mean imports? dcat remains; Elisa told us to remove prov as an import since it has reasoning issues; SKOS we're not actually referencing

Reply:
No I meant the `dct:contributor` statements, there was also a reference to UBS which was cool